.github/workflows/azure-sdk-tools.yml

```diff
@@ -0,0 +1,33 @@
+name: Test Azure SDK Tools
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
Do you intend to have this trigger on all branches? I think that is what happens if the list is empty.

Reply:
Yeahhhh. I was hoping that would disable. I wanted this not to trigger on push. I'll adjust.